components/pixels/snapshot.rs

```diff
@@ -37,20 +72,18 @@ impl Default for SnapshotAlphaMode {
 }
 
 impl SnapshotAlphaMode {
-    pub const fn is_premultiplied(&self) -> bool {
+    /// Returns None if it's opaque (can be considered as or not as premultiplied)
```

Comment:
This documentation seems out of date with the current implementation. It never returns `None`.